assets/app/views/main.js

```diff
@@ -34,8 +37,13 @@ var AppView = Backbone.View.extend({
     $('.alert-container').html('');
   },
   setAlert: function (text) {
+    var newAlert = $('<div/>', {
+      class: 'usa-alert usa-alert-error',
+      role: 'alert'
```

Comment:
Nice! I didn't know about this jQuery API but it seems like a much cleaner way to generated HTML elements 👍